### ISSUE TITLE
Playlists validation

### ISSUE THREAD

**Issue by jmrunge** [inaes-tic/mbc-mosto#13]
Mosto should validate every playlist it loads against the playlists already loaded using a library defined by Caspa in order to emit warning and error messages via pub/sub.

**Comment by fabriciocosta** [comment]
I'm ok with that, we need fps and frame length to be able to make the comparison.

**Comment by Lacrymology** [comment]
what's the difference between fps and frame length?

oh, you mean the length of the video in frames?

**Comment by fabriciocosta** [comment]
Yes. I can calculate frames length with a length formatted like that "01:35:58.33" with his "fps" as "25,001" for example. I can also do a validation or check in mosto as the server_playlist returned by getServerPlaylist give me this information. It will be nice to have it from Caspa, and make Mosto just check it. Is it possible?

**Comment by xaiki** [comment]
Forma now get it from melted

**Comment by Lacrymology** [comment]
I've been thinking about this, and I'm not sure if it can be done. Melted gets the clip length from the XMLs, and the XMLs are built from the DB data. The bottom line is, we don't have a way to double-check this from melted.

So is this issue valid?

**Comment by xaiki** [comment]
yep, if we need to hack melt/melted, that's not an issue for me, we already
know that it makes heavy processing on load (cpu).

On 17 September 2013 17:06, Tomas Neme notifications@github.com wrote:

> I've been thinking about this, and I'm not sure if it can be done. Melted
> gets the clip length from the XMLs, and the XMLs are built from the DB
> data. The bottom line is, we don't have a way to double-check this from
> melted.
> 
> So is this issue valid?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/inaes-tic/mbc-mosto/issues/13#issuecomment-24617921
> .

## 

Niv Sardi

**Comment by Lacrymology** [comment]
DTISC

I just checked, and whatever you mark as the length of the clip in the XML, the result of `list u0` says the actual length of the clip

I just created an XML that reports that the clip out should be 35474 (len + 10) and `list u0` reports

```
0 "/test/foooo.xml" 0 35464 35265 35265 25.00
```

**Comment by xaiki** [comment]
Great, so we can actually test.

## 

Niv Sardi, mobile.
